src/include/sof/audio/kpb.h

```diff
@@ -117,6 +117,13 @@ struct hb {
 	struct hb *next; /**< next history buffer */
 };
 
+struct dd {
```

Comment:
dd ? draining_data ?

Reply:
Yes, dd stands for draining data. Every instance is like struct dd draining_data; so it should be obvious for everybody who reads the code.

Reply:
Yea, it may be obvious for someone who knows the code, but anyway it's better to have a bit longer names. Especially when it's in *public* header, it would be more acceptable if it was private struct in some c file. Please use longer names in the future, for now I can live with that because we need this PR ;p